tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2995,19 +2995,41 @@ public function searchResultShouldContainSpace(string $user, string $spaceName):
 	}
 
 	/**
-	 * @Then /^for user "([^"]*)" the response should contains the parent "([^"]*)" from (?:space|mountpoint) "([^"]*)"$/
+	 * @Then /^for user "([^"]*)" the REPORT response should contains a (?:space|mountpoint) "([^"]*)" with these key and value pairs:$/
 	 *
 	 * @param string $user
-	 * @param string $parent
 	 * @param string $space
+	 * @param TableNode $table
 	 *
 	 * @return void
 	 * @throws GuzzleException
 	 */
-	public function responseShouldContainParent(string $user, string $parent, string $space): void {
-		// get a response after a Report request (called in the core)
-		$responseArray = json_decode(json_encode($this->featureContext->getResponseXml()->xpath("//d:response/d:propstat/d:prop/oc:file-parent")), true, 512, JSON_THROW_ON_ERROR);
-		Assert::assertNotEmpty($responseArray, "search result is empty");
-		Assert::assertEquals($this->getFolderId($user, $space, $parent), $responseArray[0][0], 'wrong file-parentId');
+	public function reportResponseShouldContain(string $user, string $space, TableNode $table): void {
+		$this->featureContext->verifyTableNodeColumns($table, ['key', 'value']);
+		$xmlRes = $this->featureContext->getResponseXml();
+		$resourceType = json_decode(json_encode($xmlRes->xpath("//d:response/d:propstat/d:prop/d:getcontenttype")), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
I think something like this also should work:
```suggestion
		$resourceType = $xmlRes->xpath("//d:response/d:propstat/d:prop/d:getcontenttype")[0]->__toString();
```

Reply:
Thanks, It works